analyzer-models/src/main/java/ffb/analyzer/models/espn/TeamRecordBase.java

```diff
@@ -0,0 +1,60 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class TeamRecordBase extends EspnEntity<TeamRecordBase> {
+    public enum StreakType {
+        WIN,
+        LOSS,
+        TIE,
+        NONE
+    }
+
+    private int wins;
+    private int losses;
+    private int ties;
+
+    private int gamesBack;
+
+    @JsonProperty("percentage")
+    private float winPercentage;
```

Comment:
@david-hu23 similar to the full name question - is this worth keeping since it's just a calculation of wins / total games played, all of which we can calculate? I think we need to make a more or less project wide decision on things like this. Should we take all of these cases where a field is a simple calculation of other fields and convert them into methods?

Reply:
Created #18. For the purposes of getting this in, keep it. It's simple enough to take these out.